include/libsemigroups/schreier-sims.hpp

```diff
@@ -363,6 +375,88 @@ namespace libsemigroups {
       return this->to_external_const(_strong_gens.at(depth, index));
     }
 
+    //! Get a transversal element.
+    //!
+    //! \param depth the depth.
+    //! \param pt the image of the base point under the traversal.
+    //!
+    //! \returns
+    //! A const reference to the transversal element of \c this at depth
+    //! \p depth moving the corresponding basepoint to the point \p pt.
+    //!
+    //! \throws LibsemigroupsException if the \p depth is out of bounds.
+    //! \throws LibsemigroupsException if \p pt is not in the orbit of the
+    //! basepoint.
+    //!
+    //! \complexity
+    //! Constant.
+    const_element_reference transversal_element(index_type depth,
+                                                point_type pt) {
+      if (depth >= _base_size) {
+        LIBSEMIGROUPS_EXCEPTION(
+            "depth out of bounds, expected value in range [0, %d), got %d",
+            _base_size,
+            depth);
+      }
+      if (!_orbits_lookup[depth][pt]) {
+        LIBSEMIGROUPS_EXCEPTION(
+            "no element maps %d to %d at depth %d", _base[depth], pt, depth);
+      }
+      return this->to_external_const(_transversal[depth][pt]);
+    }
+
+    //! Get an inversal element.
+    //!
+    //! \param depth the depth.
+    //! \param pt the point to map to the base point under the inversal.
+    //!
+    //! \returns
+    //! A const reference to the insversal element of \c this at depth
+    //! \p depth moving the corresponding point \p pt to the basepoint.
+    //!
+    //! \throws LibsemigroupsException if the \p depth is out of bounds.
+    //! \throws LibsemigroupsException if \p pt is not in the orbit of the
+    //! basepoint.
+    //!
+    //! \complexity
+    //! Constant.
+    const_element_reference inversal_element(index_type depth, point_type pt) {
+      if (depth >= _base_size) {
+        LIBSEMIGROUPS_EXCEPTION(
+            "depth out of bounds, expected value in range [0, %d), got %d",
+            _base_size,
+            depth);
+      }
+      if (!_orbits_lookup[depth][pt]) {
+        LIBSEMIGROUPS_EXCEPTION(
+            "no element maps %d to %d at depth %d", _base[depth], pt, depth);
+      }
+      return this->to_external_const(_inversal[depth][pt]);
+    }
+
+    //! Check if a point is in the orbit of a basepoint.
+    //!
+    //! \param depth the depth.
+    //! \param pt the point.
+    //!
+    //! \returns
+    //! A boolean indicating if the point \p pt is in the orbit of the
+    //! basepoint of \c this at depth \p depth.
+    //!
+    //! \throws LibsemigroupsException if the \p depth is out of bounds.
+    //!
+    //! \complexity
+    //! Constant.
+    bool orbits_lookup(index_type depth, point_type pt) {
```

Comment:
I think you've added documentation here to things that weren't previously documented, right? So, please make sure they actually show up in the doc when it's compiled, you probably need to add some entries in the file `docs/yml/schreiersims.yml` to make that happen.